src/context/filterContext.js

```diff
@@ -0,0 +1,25 @@
+import { createContext, useContext, useReducer } from "react";
+import {  useSearchParams } from "react-router-dom";
+import { initialFilterState } from "../reducers/constants/initialFilterState";
+import { filterReducer } from "../reducers/reducerFunctions/filterReducer";
```

Comment:
use can use absolute path instead of relative path.

See this: https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i
it will help a lot

Reply:
thanks will try this